scripts/calculateRevenue/protocols/utils/networks.ts

```diff
@@ -0,0 +1,59 @@
+import { QueryResponse, TransactionField } from '@envio-dev/hypersync-client'
+import { getHyperSyncClient } from '../../../utils'
+import { NetworkId } from '../../../types'
+
+export async function fetchTotalGasUsed({
+  networkId,
+  users,
+  startBlock,
+  endBlock,
+}: {
+  networkId: NetworkId
+  users: string[]
+  startBlock?: number
```

Comment:
Do we ever expect either of `startBlock` or `endBlock` to be `undefined`? It seems like accidentally setting these to undefined could make the query balloon in size by a bunch.

Reply:
Hypersync can handle it: [Get All ERC-20 Transfers for an Addres](https://docs.envio.dev/docs/HyperSync/hypersync-curl-examples#get-all-erc-20-transfers-for-an-address).